client/rpcserver/handlers.go

```diff
@@ -768,4 +801,49 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + logoutStr + `"`,
 	},
+	orderBookRoute: {
+		argsShort:  `"host" base quote nOrders`,
+		cmdSummary: `Retrieve all orders for a market.`,
+		argsLong: `Args:
+    host (string): The DEX to retrieve the order book from.
+    base (int): The BIP-44 coin index for the market's base asset.
+    quote (int): The BIP-44 coin index for the market's quote asset.
+    nOrders (int): Optional. Default is 0. The number of orders from the top of
+      buys and sells to return. 0 returns all orders. Epoch orders are not
```

Comment:
Please put "Default is 0." and "0 returns all orders" the same sentence, otherwise the reader will have a brief "huh?" moment wondering why zero would be the default.